app/src/main/java/com/gruutnetworks/gruutsigner/util/AuthCertUtil.java

```diff
@@ -293,7 +391,7 @@ public boolean verifyData(byte[] input, String signatureStr, String certificatio
      * @param signatureStr The signature provided for the data.
      * @return A boolean value telling you whether the signature is valid or not.
      */
-    public boolean verifyData(String input, String signatureStr)
+    private boolean verifyData(String input, String signatureStr)
```

Comment:
```suggestion
    private boolean verifySelf(String input, String signatureStr)
```
나에게 온 메시지를 검증한다는 걸 무어라 표현하는게 좋을까. 이 정도면 문맥상 자연스러울까요?
이름짓는게 어렵죠